schema_editor/app/scripts/views/recordtype/add-controller.js

```diff
@@ -2,15 +2,27 @@
     'use strict';
 
     /* ngInject */
-    function RTAddController() {
+    function RTAddController($state, RecordTypes) {
         var ctl = this;
         initialize();
 
         function initialize() {
+            ctl.recordType = {};
+            ctl.submitForm = submitForm;
+        }
 
+        /*
+         * Creates the record type and switches to the list view on success
+         */
+        function submitForm() {
+            RecordTypes.create(ctl.recordType, function() {
+                $state.go('rt.list');
+            }, function(error) {
+                console.error('Error while adding recordType: ', error);
```

Comment:
Change all `console.error` calls to `$log.debug` and inject `$log` where necessary. There is a global flag in app.js that sets the $logProvider debug mode to true when the app is provisioned for development.